Makefile

```diff
@@ -381,6 +388,9 @@ PKGS_DOCKER_LOAD=mkconf mkimage-iso-efi mkimage-raw-efi mkverification-raw-efi m
 # these packages should exists for HOSTARCH as well as for ZARCH
 # alpine-base, alpine and cross-compilers are dependencies for others
 PKGS_HOSTARCH=alpine-base alpine cross-compilers $(PKGS_DOCKER_LOAD)
+# Package dependencies auto-generated by the build system
+include pkg-deps.mk
```

Comment:
Does make ignore this in the cases where the file has not (yet) been generated? Or does it cause an error?

Reply:
Yes and no. If it doesn't exists, make will continue to process the file without any error, eventually the target for the pkg-deps.mk will be reached and executed, generating the file. At the end, make will retry to process this file, being successfully on its second pass.

Here is the full explanation from [make's manual](https://www.gnu.org/software/make/manual/html_node/Automatic-Prerequisites.html): 

```
If an included makefile cannot be found in any of these directories it is not an immediately fatal error; processing of
the makefile containing the include continues. Once it has finished reading makefiles, make will try to remake any
that are out of date or don’t exist. See [How Makefiles Are Remade](https://www.gnu.org/software/make/manual
/html_node/Remaking-Makefiles.html). Only after it has failed to find a rule to remake the makefile, or it found a rule
 but the recipe failed, will make diagnose the missing makefile as a fatal error.
```

Reply:
Would it make sense to add your explanation as a  comment in the Makefile?

Reply:
I don't think so, the comment already says it's an auto generated file and this is how make works, include directive is widely used in many projects without further explanations, I think we are good here...